frontend/testing/playwright/helpers/RouterRoute.ts

```diff
@@ -22,7 +22,7 @@ const routerRoutes: RouterRoutes = {
   dashboardCreateApp: '/dashboard/self/new',
   deploy: '/editor/{{org}}/{{app}}/deploy',
   editorOverview: `/editor/{{org}}/{{app}}/overview`,
-  editorDatamodel: `/editor/{{org}}/{{app}}/datamodel`,
+  editorDataModel: `/editor/{{org}}/{{app}}/datamodel`,
```

Comment:
Should we change this too?
```suggestion
  editorDataModel: `/editor/{{org}}/{{app}}/data-model`,
```